lib/commands/shell.js

```diff
@@ -0,0 +1,27 @@
+import log from '../logger';
+import _ from 'lodash';
+
+let commands = {};
+
+commands.mobileShell = async function (opts = {}) {
+  if (!this.relaxedSecurityEnabled) {
+    log.errorAndThrow(`Appium server must have relaxed security flag set in order to run any shell commands`);
```

Comment:
Should this mention the actual name of the flag?

Reply:
the name is anyway mentioned in the documentation. Plus less work for us if we decide to change argument name